src/applications/model/bulk-send-application.cc

```diff
@@ -208,20 +240,18 @@ BulkSendApplication::StopApplication() // Called at time specified by Stop
     }
 }
 
-// Private helpers
-
 void
 BulkSendApplication::SendData(const Address& from, const Address& to)
```

Comment:
is the idea here to transmit m_maxBytes each time this function is called to simulate a flow? If yes, why do we expect the application the produce flows of the same size? Do you plan to schedule calls to this function to simulate flows?

Reply:
The way I'm doing it is, for each generated flow, I'd schedule a callback at the given time. Inside the callback, I first set the m_maxBytes variable with the desired flow size, and then call SendData. I've since changed it to create a new BulkSendApplication::SendSize(uint32_t size) to send the desired number of bytes whenever its called.